frontend/pages/admin/UserManagementPage/components/CreateUserModal/CreateUserModal.tsx

```diff
@@ -63,6 +66,7 @@ const CreateUserModal = ({
           isModifiedByGlobalAdmin={isModifiedByGlobalAdmin}
           currentTeam={currentTeam}
           isNewUser
+          dispatch={dispatch}
```

Comment:
I'm unsure why dispatch is being passed as a prop. From what I can tell, that's not a prop that exists on `Modal.tsx`.